modular_skyrat/modules/sec_haul/code/security_medic/secmed_clothes.dm

```diff
@@ -29,6 +29,14 @@
 	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bottle, /obj/item/paper, /obj/item/melee/baton/telescopic, /obj/item/soap, /obj/item/tank/internals/emergency_oxygen, /obj/item/gun, /obj/item/storage/medkit)
 
 /obj/item/clothing/under/rank/security/peacekeeper/security_medic
+	name = "security medics's uniform"
+	desc = "A lightly armored uniform worn by Nanotrasen's Asset Protection Medical Corps."
```

Comment:
While we're here, should we change this "NAPMC" thing to something else? Security isn't asset protection, nor are they strictly part of an NT Corps.
Just something to ponder

Reply:
Can do later